charts/nri-bundle/values.yaml

```diff
@@ -26,10 +26,20 @@ newrelic-logging:
 newrelic-pixie:
   # newrelic-pixie.enabled -- Install the [`newrelic-pixie`](https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie)
   enabled: false
+  # newrelic-pixie.apiKey -- The [Pixie API key](https://docs.pixielabs.ai/reference/admin/api-keys/#create-an-api-key), generated by the New Relic guided installer
+  apiKey: ""
 
 pixie-chart:
   # pixie-chart.enabled -- Install the [`pixie-chart` chart](https://docs.pixielabs.ai/installing-pixie/install-schemes/helm/#3.-deploy)
   enabled: false
+  # pixie-chart.clusterName -- The cluster name for the Kubernetes cluster. Set this to match the value in `global.cluster`.
+  clusterName: ""
```

Comment:
Asking the user to set two properties to the same value does not seem like the best possible UX. Wouldn't it be better if the pixie chart read `global.cluster` directly?

Reply:
Thanks @roobre I will look into that and update this branch.

Reply:
Based on [Helm docs](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/), it seems like `pixie-operator-chart/templates/04_vizier.yaml` template can use this logic:

```
{{- if .Values.global.cluster }}
clusterName: {{ .Values.global.cluster }}
{{- else if .Values.clusterName }}
clusterName: {{ .Values.clusterName }}
{{- end }}
```

I'll follow up with Pixie about that part.

Reply:
https://github.com/pixie-io/pixie/pull/674